README.md

```diff
@@ -94,7 +94,7 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [comming soon]()
+- [Live Demo Link](https://book-store-8g76.onrender.com/)
```

Comment:
- [ ] OPTIONAL
Contents displayed on your deployed site appears to be obsolete. Kindly unsure to update it accordingly.